mmf/trainers/core/training_loop.py

```diff
@@ -22,13 +22,7 @@ class TrainerTrainingLoopMixin(ABC):
     num_updates: int = 0
 
     def training_loop(self) -> None:
-        self.max_updates = self.training_config.max_updates
-        self.max_epochs = self.training_config.max_epochs
-        if self.max_epochs is None:
-            self.max_epochs = math.inf
-        else:
-            self.max_updates = math.inf
-
+        self.max_updates = self._calculate_max_updates()
```

Comment:
We should keep max_epochs and max_updates mutually exclusive. So, throw a runtime error in case both are present.

Reply:
Talked offline, and decided to add a warning message when both are specified. But favoring max_epochs over max_updates when both are present.